src/components/invite/Invite.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react'
+import { Box, Grid, Typography } from '@mui/material'
+import { makeStyles } from 'tss-react/mui'
+import theme from '../../styles/createTheme'
+
+const Invite = () => {
+  const { classes } = useStyles()
+
+  return (
+    <Grid container spacing={3}>
+      <Grid item xs={12}>
+        <Typography variant="h1" className={classes.title}>
+          Invite
+        </Typography>
+        <Box
+          className={classes.imageWrapper}
+          component="img"
+          src="/img/invite/qr-invite.svg"
+        ></Box>
+        <Typography variant="body2" className={classes.description}>
+          Feel free to invite someone, who can be interesting in finding friends
+        </Typography>
+      </Grid>
+    </Grid>
+  )
+}
+
+export default Invite
+
+const useStyles = makeStyles()(() => {
+  return {
+    title: {
+      textAlign: 'center',
+      paddingBottom: '50px',
+      paddingTop: '60px',
+      [theme.breakpoints.down('sm')]: {
+        maxWidth: '290px',
+        marginLeft: 'auto',
+        marginRight: 'auto',
+      },
+    },
+    description: {
+      textAlign: 'center',
+      paddingTop: '60px',
+      [theme.breakpoints.up('md')]: {
+        textAlign: 'left',
```

Comment:
Are you sure that the text for tablet and desktop should be left-aligned

Reply:
The same style has been used as in design "Web version. Likes list". (An in design of others error pages)

Reply:
You are right.